app/views/wishlists/index.html.erb

```diff
@@ -36,14 +36,14 @@
 
               <%= link_to t("edit"), edit_wishlist_path(wishlist), class: "btn btn-sm btn-default" -%>
 
-              <a class="btn btn-sm btn-danger-outline btn-primary" data-toggle="popover" data-placement="left"
-                  data-trigger="focus" tabindex="0"
-                  data-content-source="#wlist-<%= wishlist.id -%>-delete">
+              <a class="btn btn-sm btn-danger-outline btn-primary" data-toggle="popover-with-html"
+                  data-placement="left" data-trigger="focus" tabindex="0"
```

Comment:
it should be align (indent) at the same level as element above (so no extra space) just the same level as class